backend/app/services/subscription/notification_service.py

```
@@ -0,0 +1,402 @@
+            user_id: User ID to check bindings
+
+        Returns:
+            List of channel info with binding status
+        """
+        # Get all enabled Messager channels
+        channels = (
+            db.query(Kind)
+            .filter(
+                Kind.kind == MESSAGER_KIND,
+                Kind.user_id == MESSAGER_USER_ID,
+                Kind.is_active == True,
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)